openslides/assignment/models.py

```diff
@@ -44,35 +56,87 @@ def get_root_rest_element(self):
 class Assignment(RESTModelMixin, SlideMixin, AbsoluteUrlMixin, models.Model):
     slide_callback_name = 'assignment'
 
-    STATUS = (
-        ('sea', ugettext_lazy('Searching for candidates')),
-        ('vot', ugettext_lazy('Voting')),
-        ('fin', ugettext_lazy('Finished')),
+    PHASE_SEARCH = 1
+    PHASE_VOTING = 2
+    PHASE_FINISHED = 3
+
+    PHASES = (
+        (PHASE_SEARCH, ugettext_lazy('Searching for candidates')),
+        (PHASE_VOTING, ugettext_lazy('Voting')),
+        (PHASE_FINISHED, ugettext_lazy('Finished')),
     )
 
-    name = models.CharField(max_length=100, verbose_name=ugettext_lazy("Name"))
-    description = models.TextField(null=True, blank=True, verbose_name=ugettext_lazy("Description"))
-    posts = models.PositiveSmallIntegerField(verbose_name=ugettext_lazy("Number of available posts"))
+    title = models.CharField(
+        max_length=100,
+        verbose_name=ugettext_lazy("Title"))
+    """
+    Title of the assignment.
+    """
+
+    description = models.TextField(
+        blank=True,
+        verbose_name=ugettext_lazy("Description"))
+    """
+    Text to describe the assignment.
+    """
+
+    open_posts = models.PositiveSmallIntegerField(
+        verbose_name=ugettext_lazy("Number of members that should be elected"))
```

Comment:
Maybe use: `"Number of members to be elected"` or `"Number of members that are to be elected"` so that it sounds less optional.
The same in the following docstring.